config/300-buildpack-clusterbuildtemplate.yaml

```diff
@@ -42,22 +42,30 @@ spec:
     description: When set, skip the detect step and use the given buildpack.
     default: ''
   steps:
-  - args:
-    - -c
-    - |
-      chown -R "${USER_ID}:${GROUP_ID}" "/builder/home" \
-        && chown -R "${USER_ID}:${GROUP_ID}" /layers \
-        && chown -R "${USER_ID}:${GROUP_ID}" /workspace
-    command:
-    - /bin/sh
-    image: alpine
+  - name: info
+    image: github.com/google/kf/cmd/buildinfo
     imagePullPolicy: Always
-    name: prepare
-    resources: {}
+    args:
+    - "--uid=${USER_ID}"
+    - "--gid=${GROUP_ID}"
+    - "--app=/workspace"
+    - "--image=${IMAGE}"
+    - "--run-image=${RUN_IMAGE}"
+    - "--builder-image=${BUILDER_IMAGE}"
+    - "--use-cred-helpers=${USE_CRED_HELPERS}"
+    - "--cache=${CACHE}"
+    - "--buildpack=${BUILDPACK}"
     volumeMounts:
-    - mountPath: /layers
-      name: ${CACHE}
-  - args:
+    - name: "layers-dir"
```

Comment:
Should "layers-dir" be hardcoded or should this be a variable?

Reply:
Tekton hard-codes this because it's a volume that has to be shared container to container, but should be reaped with the pod. We could allow users to change it to keep the volume around after a build, but I think that introduces quite a bit of extra complexity.

Reply:
Sounds good. Let's not add the complexity.